ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs

```diff
@@ -1098,5 +1099,15 @@ public FrameworkIndex GetFrameworkIndex()
         {
             return _frameworks;
         }
+
+        private static readonly Regex indexerRegex = new Regex(@"\[.*?\]$", RegexOptions.Compiled);
+        private static string RemoveindexerFromPropertyName(string str)
+        {
+            if (string.IsNullOrEmpty(str))
+            {
+                return str;
+            }
+            return indexerRegex.Replace(str, "[]");
```

Comment:
no need for regex, we can just do something like:
```suggestion
            var leftIndex = str.IndexOf('[');
if (leftIndex > 0)
{
    return str.Substring(0, leftIndex) + "[]";
}
return str;
```